### ISSUE TITLE
Yabai focus is slow or non-existent in some scenarios

### ISSUE THREAD

**Issue by alber70g** [koekeishiya/yabai#1958]
I'm using the following config for focus:

```
yabai -m config mouse_follows_focus           on           # on, off
yabai -m config focus_follows_mouse           autoraise    # off, autoraise, autofocus
```

When switching my mouse to another window it should get focus. However, most of the time, when I move the cursor and stop, while hovering over the window, it doesn't focus _until_ I move the cursor only a tiny bit.

I've added a video to demonstrate this

https://github.com/koekeishiya/yabai/assets/516972/7b36f765-e8b3-4ba5-baeb-6139c4a9668e

I'd like it to be instant, or at least focus without having to move sloowwly or re-move to get focus. It used to be better, as I remember.

**Comment by koekeishiya** [comment]
This seems to be caused by some kind of mouse acceleration interference or something. yabai is reacting to mouse_moved events, and looking at event logs, the cursor is actually moving after yabai has received the last mouse_moved event that it can react to.

Closing as not fixable.

**Comment by koekeishiya** [comment]
Nvm.. you are correct. This is caused by the logic implemented to try and reduce the number of events we respond to:

```
    uint64_t event_time = CGEventGetTimestamp(context);
    float dt = ((float) event_time - g_mouse_state.last_moved_time) * (1.0f / 1E6);
    if (dt < 25.0f) goto out;
```

I have look at cpu usage with and without and the difference is very small, so I will remove this in favor of increased responsiveness.

**Comment by koekeishiya** [comment]
Fixed on master.

**Comment by koekeishiya** [comment]
Fixed in v7.0.0.
